packages/esm-patient-chart-app/src/banner/patient-banner.component.tsx

```diff
@@ -0,0 +1,130 @@
+    [patientUuid, onClick, onTransition],
+  );
+
+  const patientName = `${patient.name?.[0].given?.join(' ')} ${patient?.name?.[0].family}`;
+  const patientPhotoSlotState = React.useMemo(() => ({ patientUuid, patientName }), [patientUuid, patientName]);
+
+  const [showContactDetails, setShowContactDetails] = React.useState(false);
+  const toggleContactDetails = React.useCallback((event: MouseEvent) => {
+    event.stopPropagation();
+    setShowContactDetails((value) => !value);
+  }, []);
+
+  const patientAvatar = (
+    <div className={styles.patientAvatar} role="img">
+      <ExtensionSlot extensionSlotName="patient-photo-slot" state={patientPhotoSlotState} />
+    </div>
+  );
+
+  const handleNavigateToPatientChart = (event: MouseEvent) => {
+    if (onClick) {
+      !(overFlowMenuRef?.current && overFlowMenuRef?.current.contains(event.target)) && onClick(patientUuid);
+    }
+  };
+  const [showDropdown, setShowDropdown] = React.useState(false);
+  const closeDropdownMenu = React.useCallback((event: MouseEvent) => {
+    event.stopPropagation();
+    setShowDropdown((value) => !value);
+  }, []);
+
+  return (
+    <div className={styles.container} role="banner">
+      <div
+        onClick={handleNavigateToPatientChart}
+        tabIndex={0}
+        role="button"
+        className={`${styles.patientBanner} ${onClick && styles.patientAvatarButton}`}
+      >
+        {patientAvatar}
+        <div className={styles.patientInfo}>
+          <div className={`${styles.row} ${styles.patientNameRow}`}>
+            <div className={styles.flexRow}>
+              <span className={styles.patientName}>{patientName}</span>
+              <ExtensionSlot
+                extensionSlotName="patient-banner-tags-slot"
+                state={{ patientUuid, patient }}
+                className={styles.flexRow}
+              />
+            </div>
+            {!hideActionsOverflow && (
+              <div ref={overFlowMenuRef}>
+                <CustomOverflowMenuComponent
+                  menuTitle={
+                    <>
+                      <span className={styles.actionsButtonText}>{t('actions', 'Actions')}</span>{' '}
+                      <OverflowMenuVertical size={16} style={{ marginLeft: '0.5rem' }} />
+                    </>
+                  }
+                  dropDownMenu={showDropdown}
+                >
+                  <ExtensionSlot
+                    onClick={closeDropdownMenu}
+                    extensionSlotName="patient-actions-slot"
+                    key="patient-actions-slot"
+                    className={styles.overflowMenuItemList}
+                    state={patientActionsSlotState}
+                  />
+                </CustomOverflowMenuComponent>
+              </div>
+            )}
+          </div>
+          <div className={styles.demographics}>
+            <span>{capitalize(patient.gender)}</span> &middot; <span>{age(patient.birthDate)}</span> &middot;{' '}
```

Comment:
I don't think `lodash.capitalize` respects locales, so this might give a poor conversion for other languages.
TODO: Add capital gender to translation file.

Reply:
I have done the translation I don't know if I have done it right.